.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: objective-c
+osx_image: xcode8
+
+before_install:
+- xcrun simctl list # list devices
```

Comment:
Just for fun? Or does this have funky side effects that make Xcode more willing to setup a sim?